monai/networks/nets/controlnet.py

```diff
@@ -34,84 +34,71 @@
 from collections.abc import Sequence
 
 import torch
-import torch.nn.functional as F
 from torch import nn
 
 from monai.networks.blocks import Convolution
 from monai.networks.nets.diffusion_model_unet import get_down_block, get_mid_block, get_timestep_embedding
 from monai.utils import ensure_tuple_rep
 
 
-class ControlNetConditioningEmbedding(nn.Module):
+class ControlNetConditioningEmbedding(nn.Sequential):
     """
     Network to encode the conditioning into a latent space.
     """
 
     def __init__(self, spatial_dims: int, in_channels: int, out_channels: int, channels: Sequence[int]):
-        super().__init__()
-
-        self.conv_in = Convolution(
-            spatial_dims=spatial_dims,
-            in_channels=in_channels,
-            out_channels=channels[0],
-            strides=1,
-            kernel_size=3,
-            padding=1,
-            conv_only=True,
-        )
-
-        self.blocks = nn.ModuleList([])
-
+        convs = [
```

Comment:
Will this name changing here influence the weight loading?

Reply:
hm i just checked and it does. i could implement `load_old_state_dict` here, but maybe it isn't worth placing an extra burden on users when the gains are quite small: just a slight reduction in the amount of code, rather than any functional differences. what do you think?

Reply:
Yes, no need to such a small update, but do we have a `load_old_state_dict` for `ControlNet`?

Reply:
I would suggest against this change but also add activation layer in the `Convolution` which can also make code cleaner. What do you think?
cc @ericspod

Reply:
sorry i have done some tests and realised we do need to add a `load_old_state dict`. i will add it + a test for compatibility. I'll also make the suggested change with the activation layer